src/strategies/npm-audit.js

```diff
@@ -14,18 +23,60 @@ export function NPMAuditStrategy() {
   };
 }
 
+async function checkIfProjectUsePnpm(path) {
+  return new Promise((resolve) => {
+    fs.access(`${path}/pnpm-lock.yaml`, fs.constants.F_OK, (err) => {
+      if (err) {
+        resolve(false);
+      } else {
+        console.log("Project use pnpm");
+        resolve(true);
+      }
+    });
+  });
+}
```

Comment:
Why not using the Asynchronous method instead of re-implementing a Promise by hand?